tests/test_path.nim

```diff
@@ -0,0 +1,30 @@
+# Nim-LibP2P
+# Copyright (c) 2023-2025 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0 ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+{.used.}
+
+import std/os
+import ./imports
+
+const test_path {.strdefine.} = ""
+
+when test_path == "":
+  {.
+    error:
+      "Please specify test_path via -d:test_path=\"path/to/test\" (e.g., -d:test_path=\"quic\")"
```

Comment:
consider `-d:test_path` -> `-d:path` 

since there are no any other `path` arguments, consider using just that to make command bit shorter and bit easier remember.